WcaOnRails/spec/mailers/previews/wca_monthly_digest_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/wca_monthly_digest_mailer
+class WcaMonthlyDigestMailerPreview < ActionMailer::Preview
+  def send_weat_digest_content
+    mail = nil
+    ActiveRecord::Base.transaction do
+      mail = WcaMonthlyDigestMailer.send_weat_digest_content
+    end
+    mail
```

Comment:
Ruby has implicit returns. You should be able to do something like

```ruby
def send_weat_digest_content
  ActiveRecord::Base.transaction do
    WcaMonthlyDigestMailer.send_weat_digest_content
  end
end
```

as per https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/DatabaseStatements.html#method-i-transaction, where it say `[...] and returns the result of the block`.

On a more high-level sidenote, why are you even wrapping this into a transaction? You're not writing anything.